translations/messages.en.yaml

```diff
@@ -4,6 +4,8 @@ ngsite.layout.search_site: "Search %site_name%"
 ngsite.layout.site_frontpage: "%site_name% front page"
 ngsite.layout.meta_generator: "Ibexa DXP powered by Netgen"
 ngsite.layout.powered_by: "Powered by"
+ngsite.layout.cookie_banner: "Cookie Banner"
```

Comment:
`Cookie banner` (lowercase `b`)

Reply:
If that's the case, we will fix it when it comes up. Otherwise, it just looks weird.